### PR TITLE
Feature/argodev 512

### PR DESCRIPTION
**Pull request by vidkris**
Class to compute doxy from P

### CODE REVIEW

File: bgc_derivation/p2doxy.m [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    function a=A(this,temp,psal,Spreset)
+      a=(1013.25-ph2ocalc(this,temp,Spreset))./(1013.25-ph2ocalc(this,temp,psal));
+    end
+
+    % calculate pH2O
+    function ph2o=ph2ocalc(this,temp,salin)
+      ph2o=1013.25*exp(this.D0+this.D1*(100./(temp+273.15))+ ...
+                       this.D2*log((temp+273.15)./100)+this.D3*salin);
+    end
+    %generate a matrix of ceratin coeff
```
Comment (qtl-bodc) [inline bgc_derivation/p2doxy.m:114]: spelling 'ceratin'

File: bgc_derivation/p2doxy.m [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+      tpot=tetai(pres,temp,psal,0);
+      [null,sigma0]=swstat90(psal,tpot,0);
+      rho=(sigma0+1000)/1000;
+
+      % compute doxy [umol/kg]
+      oxy=molardoxycalc(this,dphase,pres,temp).*salcorrcalc(this,psal,temp).*prescorrcalc(this,pres,temp);
+      doxy=oxy./rho';
+    end
+    
+    % calculate moalrdoxy
```
Comment (qtl-bodc) [inline bgc_derivation/p2doxy.m:79]: spelling 'moalrdoxy'

**Review by qtl-bodc — COMMENTED** [review]
Have reviewed. Just a couple of spelling errors to correct.

File: bgc_derivation/p2doxy.m (hunk 1 above)
Comment (vidkris) [reply bgc_derivation/p2doxy.m]: fixed.

File: bgc_derivation/p2doxy.m (hunk 2 above)
Comment (vidkris) [reply bgc_derivation/p2doxy.m]: fixed

**Comment by qtl-bodc** [comment]
ok, looks good to merge.

**Pull request merged**